yarn.lock

```diff
@@ -11020,6 +10877,15 @@ __metadata:
   languageName: node
   linkType: hard
 
+"enquirer@npm:^2.3.6":
```

Comment:
We already have `enquirer@npm:2.3.5` in the lockfile, perhaps we should deduplicate them